src/sumaDeArrays.ts

```diff
@@ -8,26 +8,19 @@ for (let i = 0; i < 6; i++) {
     prompt("Ingreso numero: " + (i + 1) + "\nIngrese sus 6 primeros numeros: ")
   );
   ingresoDeNumeros1[i] = num1;
+  console.log("Posicion Array1: " + i + " Numero ingresado: " + num1);
 }
-// Muestro en consola para el seguimiento de los ingresos/valores
-for (let i = 0; i < ingresoDeNumeros1.length; i++) {
-  console.log(
-    "Posicion Array1: " + i + " Numero ingresado: " + ingresoDeNumeros1[i]
-  );
-}
+
```

Comment:
No hace falta crear otro for para el console.log, podemos usar el mismo for que se usa para pedir los datos al usuario.